velox/exec/VectorHasher.cpp

```diff
@@ -587,9 +654,13 @@ void VectorHasher::cardinality(uint64_t& asRange, uint64_t& asDistincts) {
     rangeOverflow_ = true;
```

Comment:
Comment for the line above (original line 586): Should `__builtin_sub_overflow` takes the max and min values after the extension?